integreat_cms/cms/templates/feedback/_admin_feedback_filter_form.html

```diff
@@ -38,7 +38,7 @@
         </div>
     </div>
     <div class="w-full flex p-4 justify-end gap-4">
-        <a id="filter-reset" class="btn btn-gray">{% translate "Reset filter" %}</a>
+        <a id="filter-reset" class="btn btn-light-blue">{% translate "Reset filter" %}</a>
```

Comment:
```suggestion
        <a id="filter-reset" class="btn btn-ghost">{% translate "Reset filter" %}</a>
```

> I have one small suggestion (though not specified in the issue scope and not mandatory): can we maybe make the filter buttons (in Feedback list, Media library, page list, event list) look exactly like the new preview buttons (light sky blue background + blue font)?

I agree, I also understood the guideline in a way that we only have three different kind of buttons, and convert the light blue buttons to ghost buttons...